WooCommerce/Classes/ViewRelated/Orders/OrdersSplitViewWrapperController.swift

```diff
@@ -45,6 +45,8 @@ final class OrdersSplitViewWrapperController: UIViewController {
         let loaderViewController = OrderLoaderViewController(orderID: orderID, siteID: Int64(siteID), note: note)
         let loaderNavigationController = WooNavigationController(rootViewController: loaderViewController)
 
+        ordersViewController.navigationController?.popToRootViewController(animated: false)
```

Comment:
This is surprising in the context of a `presentDetails` function. 

I think it's there because of the double-details bug we've seen, but that's only because I can remember stuff from the last few days. It would probably help future-us to explain why we're popping here in a comment.